src/coreclr/src/tools/Common/JitInterface/ThunkGenerator/gen.bat

```diff
@@ -1,2 +1,2 @@
 cd /d %~dp0
-dotnet run -- ThunkInput.txt ..\CorInfoBase.cs ..\..\..\jitinterface\jitinterface.h
\ No newline at end of file
+dotnet run -- ThunkInput.txt ..\CorInfoBase.cs ..\..\..\crossgen2\jitinterface\jitinterface.h
```

Comment:
Not sure in this one but I had to change the path to make it work cc @MichalStrehovsky

Reply:
The Common directory was moved around recently. Thanks for the fix.